assignments/prototypes.js

```diff
@@ -33,15 +51,26 @@
   * should inherit takeDamage() from CharacterStats
 */
  
+function Humanoid(elem) {
+  this.team = elem.team;
+  this.weapons = elem.weapons;
+  this.language = elem.language;
+  CharacterStats.call(this, elem);
```

Comment:
A small thing, but I usually like to do the `call` first, that way I make sure it doesn't overwrite any properties I have  already set.